vars/phaseDeploy.groovy

```diff
@@ -3,21 +3,23 @@ import org.ods.service.OpenShiftService
 import org.ods.service.ServiceRegistry
 import org.ods.util.GitUtil
 import org.ods.util.MROPipelineUtil
+import org.ods.util.PipelineSteps
 import org.ods.util.Project
 
 def call(Project project, List<Set<Map>> repos) {
+    def steps = new PipelineSteps(this)
```

Comment:
@georgfederman why not use the service registry here? Would make testing easier, as right now, PipelineSteps is not exchangable with our do-nothing implementation in test/util/PipelineSteps.groovy.

@martsec FYI

Reply:
good thought! will do that.